src/docs/implementations/dynatrace.adoc

```diff
@@ -11,6 +11,52 @@ include::install.adoc[]
 
 == Configuring
 
+=== Quick setup
+
+For setting up new integrations with Dynatrace, it is recommended to use the latest version of the API (`api-version: v2`).
+
+Use the following code in your project to export Micrometer metrics to the specified Dynatrace endpoint. 
+https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/[The Dynatrace API token documentation] contains more information on how to create an API token.
+The metrics ingest (`metrics.ingest`) permission is required on the token in order to ingest metrics.
+More info on how to use the v2 API is provided <<bookmark-apiv2, below>>.
+
+[source,java]
+----
+DynatraceConfig dynatraceConfig = new DynatraceConfig() {
+    @Override
+    public DynatraceApiVersion apiVersion() {
+        return DynatraceApiVersion.V2;
+    }
+
+    @Override
+    public String apiToken() {
+        return MY_TOKEN;
+    }
+
+    @Override
+    public String uri() {
+        return MY_DYNATRACE_URI;
+    }
+
+    @Override
+    @Nullable
+    public String get(String k) {
+        return null;
+    }
+};
+MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
+----
+
+If you are working with Spring Boot, you can instead put the following properties in your `application-dynatrace.yml` in the `resources` folder.
+
+[source,yml]
+----
+management.metrics.export.dynatrace:
+    enabled: true
+    api-token: YOUR_TOKEN
+    uri: https://{your-environment-id}.live.dynatrace.com
```

Comment:
```suggestion
    api-version: v2
    uri: ${MY_DT_ENV_URI} # https://{your-environment-id}.live.dynatrace.com
    api-token: ${MY_DT_ENV_TOKEN}
```
We should not suggest placing the values directly in the YAML but refer to the environment variable placeholder mechanism (https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-placeholders-in-properties), especially for secrets.

Also URI first and token for that URI second looks more natural to me.